test_templates.sh

```diff
@@ -404,6 +404,9 @@ sed -i "/.*components\/ceph\/v[1-9][0-9]*\/.*/d" build_temp/test.pan
 # Only test default schema version of ssh component
 sed -i "/.*components\/ssh\/schema-.*/d" build_temp/test.pan
 
+# action.pan is included by the schema and can only be included after inputs.pan
```

Comment:
nitpick: can you put this at line 398 so it's co-located with the other metaconfig fix ups please?

Reply:
done